flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/JdbcDataTypeTest.java

```diff
@@ -144,7 +160,13 @@
                         "TIMESTAMP(9) WITHOUT TIME ZONE",
                         "The precision of field 'f0' is out of the TIMESTAMP precision range [1, 6] supported by PostgreSQL dialect."),
                 createTestItem(
-                        "postgresql", "TIMESTAMP_LTZ(3)", "Unsupported type:TIMESTAMP_LTZ(3)"));
+                        "postgresql", "TIMESTAMP_LTZ(3)", "Unsupported type:TIMESTAMP_LTZ(3)"),
```

Comment:
Oracle dialect supports Data Type `TIMESTAMP WITH LOCAL TIME ZONE`, and its type meaning is same with Flink SQL Data Type `TIMESTAMP_LTZ`, I think we can support it naturally.

Reply:
@leonardBang this is for postgresql.. I have to change de line because this was the last line..
but nothing change